### PR TITLE
Add way to send larger spartn corrections messages

### PR DESCRIPTION
**Pull request by btdubs**
* Brings Greenzie fork up to date with upstream
* Adds /spartn topic to forward SPARTN format corrections data to Ublox device
* Increase buffer size to handle larger SPARTN corrections messages
* Debianize

Errors seen on buffer size previously had SPARTN correction messages of size 12996 hence the increase in size. Dynamic buffer resizing is possible but probably unnecessary.

### CODE REVIEW

File: ublox_gps/src/node.cpp [hunk 1]
```diff
@@ -1813,10 +1813,15 @@ void rtcmCallback(const rtcm_msgs::Message::ConstPtr &msg) {
...
 
+void spartnCallback(const std_msgs::UInt8MultiArray::ConstPtr &msg) {
+  gps.sendSpartn(msg->data);
+}
+
 int main(int argc, char** argv) {
   ros::init(argc, argv, "ublox_gps");
   nh.reset(new ros::NodeHandle("~"));
   ros::Subscriber subRtcm = nh->subscribe("/rtcm", 10, rtcmCallback);
+  ros::Subscriber subRtcm = nh->subscribe("/spartn", 10, spartnCallback);
```
Comment (AGummyBear) [inline ublox_gps/src/node.cpp]: This is overriding the ROS subscriber for `subRtcm`

**Review by AGummyBear — CHANGES_REQUESTED** [review]
Need to fix bug with ROS subscribers.

More generally, since the data flow path for RTCM and SPARTN is fundamentally identical, should we just generalize our SPARTN subscriber to be a binary message subscriber? (Same functionality, just change the name)

File: ublox_gps/src/node.cpp (hunk 1 above)
Comment (btdubs) [reply ublox_gps/src/node.cpp]: Fixed

**Comment by btdubs** [comment]
> Need to fix bug with ROS subscribers.
> 
> More generally, since the data flow path for RTCM and SPARTN is fundamentally identical, should we just generalize our SPARTN subscriber to be a binary message subscriber? (Same functionality, just change the name)

I considered that but the [original thread that merged the RTCM](https://github.com/KumarRobotics/ublox/pull/136) changes liked having the rtcm_msg instead of generic message types so I kept it for RTCM. @AGummyBear

**Comment by AGummyBear** [comment]
> > Need to fix bug with ROS subscribers.
> > More generally, since the data flow path for RTCM and SPARTN is fundamentally identical, should we just generalize our SPARTN subscriber to be a binary message subscriber? (Same functionality, just change the name)
> 
> I considered that but the [original thread that merged the RTCM](https://github.com/KumarRobotics/ublox/pull/136) changes liked having the rtcm_msg instead of generic message types so I kept it for RTCM. @AGummyBear 

Fair enough. In that case, let's leave it as-is for SPARTN

**Comment by 1hada** [comment]
Oof , the PR is pointing to master! please remember to point it to `noetic-devel`

**Review by 1hada — APPROVED** [review]
After running on GZ-7 for about 5 minutes I have not seen any overflow messages all while still seeing and RTK fix and 0.033 [m] of horizontal accuracy.

**Pull request merged**